docs/building.md

```
@@ -0,0 +1,99 @@
+testing:
+
+```bash
+python packaging/verify_universal.py
+```
+
+There is no Poppler step on any platform. PDF rendering is inside the
+`pypdfium2` wheel and needs no system binary.
+
+## Project layout
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)